minimal-notebook/Dockerfile

```diff
@@ -70,11 +71,11 @@ RUN cd /tmp && \
     echo "9ea57c0fdf481acf89d816184f969b04bc44dea27b258c4e86b1e3a25ff26aa0 *Miniconda3-3.19.0-Linux-x86_64.sh" | sha256sum -c - && \
     /bin/bash Miniconda3-3.19.0-Linux-x86_64.sh -f -b -p $CONDA_DIR && \
     rm Miniconda3-3.19.0-Linux-x86_64.sh && \
-    $CONDA_DIR/bin/conda install --yes conda==3.19.1 && \
+    $CONDA_DIR/bin/conda install --quiet --yes conda==3.19.1 && \
```

Comment:
This is why the Miniconda install line cannot come from cache. The change is not currently in `master`.